tools/accuracy_checker/openvino/tools/accuracy_checker/representation/regression_representation.py

```diff
@@ -158,3 +158,26 @@ def value(self):
     @value.setter
     def value(self, value):
         self._value = value
+
+class HandLandmarksRepresentation(BaseRepresentation):
+    def __init__(self, identifier='', x_values=None, y_values=None):
+        super().__init__(identifier)
+        self.x_values = x_values if x_values is not None else []
+        self.y_values = y_values if y_values is not None else []
+
+
+class HandLandmarksAnnotation(HandLandmarksRepresentation):
+    @property
+    def interocular_distance(self):
```

Comment:
interocular distance is distance between eyes on face, not sure that it is good idea reuse it for hands landmarks

Reply:
fixed